multi-agent/lib/mas/elements/llms/openai/openai.py

```
@@ -125,15 +135,29 @@ def stream(
-                tool_calls=aggregator.build(),
+                tool_calls=tool_calls,
             )
 
     def bind_tools(self, tools: List[ToolDefinition]) -> OpenAILLM:
         clone = copy.copy(self)
+        clone._fwd_names, clone._rev_names = build_name_maps(
+            t.name for t in tools
+        )
         clone._tools = OpenAIToolsConverter.to_openai(tools)
+        if clone._tools:
+            for tool_param, tool_def in zip(clone._tools, tools):
```

Ruff (0.16.3)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)